src/test/scala/com/cloudera/sparkts/models/ARIMAXSuite.scala

```diff
@@ -0,0 +1,136 @@
+/**
```

Comment:
PyFlux author here - stumbled upon this! Indirect thank you for doing these comparisons; I think the difference is due to parameter differences in the optimizers. Looks like the PyFlux implementation gets stuck because the tolerance (pgtol) is quite high and accuracy (factr) is quite low - default settings for scipy.fmin_l_bfgs_b. Will have to amend for a future release (which will prevent it stopping prematurely as it did above in your comparison)